kopf/k8s/patching.py

```diff
@@ -20,20 +28,20 @@ def patch_obj(*, resource, patch, namespace=None, name=None, body=None):
     name = body.get('metadata', {}).get('name') if body is not None else name
 
     api = kubernetes.client.CustomObjectsApi()
-    if namespace is None:
-        api.patch_cluster_custom_object(
-            group=resource.group,
-            version=resource.version,
-            plural=resource.plural,
-            name=name,
-            body=patch,
-        )
-    else:
-        api.patch_namespaced_custom_object(
-            group=resource.group,
-            version=resource.version,
-            plural=resource.plural,
-            namespace=namespace,
-            name=name,
-            body=patch,
-        )
+    request_kwargs = {
+        'group': resource.group,
+        'version': resource.version,
+        'plural': resource.plural,
+        'name': name,
+        'body': patch
+    }
+    patch_func = api.patch_cluster_custom_object
+    if namespace is not None:
+        request_kwargs['namespace'] = namespace
+        patch_func = api.patch_namespaced_custom_object
+    loop = asyncio.get_event_loop()
+    if not loop:
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
```

Comment:
Same story with the excplicit loop management as in `post_event()`

Reply:
Fixed this one too.